<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoNuevaCuenta</class>
 <widget class="QDialog" name="DialogoNuevaCuenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crear Nueva Cuenta</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Nombre de la Cuenta</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Descripci√≥n</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>94</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Codigo:</string>
   </property>
   <widget class="QRadioButton" name="codAutomatico">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatico</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="codManual">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="codigoManual">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>codAutomatico</tabstop>
  <tabstop>codManual</tabstop>
  <tabstop>codigoManual</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogoNuevaCuenta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogoNuevaCuenta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
